### DISCUSSION TITLE
Log4J

### DISCUSSION THREAD

**Discussion by MariaDavidov595** [apache/airflow#20288] [General]
Is Airflow using Log4j? If so, what version of log4j Airflow is using?

**Comment by potiuk** [comment]
Worth checking if a question has not beeen asked before. Did you actually search `Log4J` before asking the question?

https://github.com/apache/airflow/discussions/20279

**Comment by MariaDavidov595** [comment]
I actually saw this answer, however the reason behind the 'No' wasn't clear. 
Airflow is not impacted by log4j because it doesn't use it? Or because log4j was already upgraded?

**Reply by potiuk** [reply]
So why did not you ask there?

**Reply by MariaDavidov595** [reply]
Because I'm new to this forum and it seemed like I can only answer the original question and not to ask additional questions

**Reply by potiuk** [reply]
Nope. You can participate in any discussion. See the answer there then. I updated it there.
